crates/pop-cli/src/commands/up/mod.rs

```diff
@@ -173,23 +168,6 @@ mod tests {
 		})
 	}
 
-	#[tokio::test]
-	#[cfg(any(feature = "polkavm-contracts", feature = "wasm-contracts"))]
-	async fn detects_contract_correctly() -> anyhow::Result<()> {
```

Comment:
Tested in integration test: https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-cli/tests/contract.rs#L116